Packs/Legacy/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py

```diff
@@ -170,6 +173,44 @@ def test_similar_context(mocker):
 
     args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'listValue.name'})
     result = main()
+    assert len(result['EntryContext']['similarIncidentList']) == 1
+    assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 2
+
+    args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'dictListValue.test'})
+    result = main()
     assert len(result['EntryContext']['similarIncidentList']) == 2
     assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 3
     assert result['EntryContext']['similarIncidentList'][1]['rawId'] == 2
+
+    args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'dictListValue'})
+    result = main()
+    assert len(result['EntryContext']['similarIncidentList']) == 1
+    assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 2
+
+
+def dt_res(context, key):
+    if key == 'simpleValue':
```

Comment:
I would prefer to implement a simple dt logic, so if someone will change the values of context, it will still work.
A simple logic can be:
split by "." and return the nested values for that. You can use on of the methods here:
https://stackoverflow.com/questions/25833613/python-safe-method-to-get-value-of-nested-dictionary
I think `safeget` or `deep_get` both will be good for you

Reply:
Yes i agree

Reply:
yes i agree

Reply:
done